src/workerd/api/actor-state.c++

```diff
@@ -210,6 +210,25 @@ kj::Promise<void> updateStorageDeletes(IoContext& context,
   metrics.addStorageDeletes(deleted);
 };
 
+// Return the id of the current actor (or the empty string if there is no current actor).
+kj::String getCurrentActorId() {
+  if (IoContext::hasCurrent()) {
+    IoContext& ioContext = IoContext::current();
+    KJ_IF_SOME(actor, ioContext.getActor()) {
+      KJ_SWITCH_ONEOF(actor.getId()) {
+	KJ_CASE_ONEOF(s, kj::String) {
+	  return kj::heapString(s);
+	}
+	KJ_CASE_ONEOF(actorId, kj::Own<ActorIdFactory::ActorId>) {
+	  return actorId->toString();
+	}
+	KJ_UNREACHABLE;
+      }
+    }
+  }
+  return kj::heapString("");
```

Comment:
Not blocking but I'm curious why returning the empty string is preferred here as opposed to using `kj::Maybe<kj::String>` and returning `kj::none`?

Reply:
Mostly because `kj::Maybe` doesn't automatically get turned into a string.  Looking harder, there's a one-liner that will do what I want, so I'm happy to make this change.

Reply:
Fixed.